content/uk/docs/reference/glossary/network-policy.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Network Policy
```

Comment:
```yaml
title: NetworkPolicy
```
?

Reply:
It was the original title. Translation is in aka.